Java server integration

DiJan opened the pull request:
RUN-78 DJ/MZ : Added file upload capability to the generator for Company images on create and edit pages.

DiJan committed:
RUN-78 made an artificial change to see if we can generate.

DiJan committed:
RUN-78 MZ/DJ  added the file upload to the template.

DiJan committed:
RUN-78 DJ/MZ Added file upload capability to Edit/Add company.

DiJan committed:
RUN-78 fixed the JavaScript error.

DiJan committed:
RUN-78 MZ/DJ: Added the image preview to the company.

DiJan committed:
RUN-78 MZ/DJ Fixed tests.

JLBoor commented on app/templates/skeleton/app/i18n/locale-fr.json:
```diff
@@ -54,13 +54,15 @@
         "ACTION_UPDATE": "Mettre à jour",
-        "ACTION_DELETE": "Supprimer"
+        "ACTION_DELETE": "Supprimer",
+        "INVALID_IMAGE_FILE": "Les fichiers doivent être les images valides , à moins de 10MB en taille."
```
Les fichiers doivent être des images valides, inférieur à 10MB.

JLBoor commented on app/templates/skeleton/app/index.html:
```diff
@@ -21,6 +21,9 @@
   <script src="../bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+  <script src="../bower_components/ng-file-upload/ng-file-upload-shim.js"></script>
+  <script src="../bower_components/ng-file-upload/ng-file-upload.js"></script>
+  <script src="../bower_components/less/dist/less.js"></script>
```
Are we using less.js in the production version?

JLBoor commented:
I believe the tests are broken. That's the stacktrace I see on TravisCI

Running "karma:unit_and_coverage" (karma) task
PhantomJS 1.9.8 (Linux) ERROR
  TypeError: 'undefined' is not a function (evaluating 'defaultFunc.eval.bind(defaultFunc)')
  at /home/travis/build/JLBoor/generator-jlb-angular/app/templates/skeleton/bower_components/less/dist/less.js:1793
Finished in 0.09 secs / 0 secs

DiJan committed:
RUN-78 MZ/DJ fixed the style in IE.

DiJan committed:
RUN-78 MZ/DJ: Corrected French Translation.

DiJan committed:
RUN-78 MZ/DJ : Took out less.js.
Also moved angular-mocks tp devDependencies.
Added a target in gruntfile.js less:dev to generate the CSS for dev.

DiJan commented:
Corrected the French translation.

Took out the less.js
moved the mock to dev-dependencies.
added a dev target to generate css from less
